src/app.jsx

```diff
@@ -483,19 +483,10 @@ class Application extends React.Component {
         client.streamEvents(system, message => this.handleEvent(message, system))
                 .catch(e => console.error("init", system ? "system" : "user", "streamEvents failed:", e.toString()))
                 .finally(() => {
+                    console.log("init", system ? "system" : "user", "podman service closed");
```

Comment:
This added line is not executed by any test.

Reply:
So I suppose we never test, cleanupAfterService now or before, but can we do that? Wouldn't it just involve `systemctl stop cockpit.socket` while the page is open?

Reply:
Ah! `testFailingPodmanService` checks exactly that, it stops both user and system service and verifies that the empty state appears. However, it does it synchronously *before* logging into cockpit. I added a test which checks it while cockpit is running, and I see these messages in the console now. So next round should be covered.